charts/template-api/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+appVersion: "1.0"
```

Comment:
Use Docker image tag of template here to avoid compatibility issues. If this chart can be used by any symfony application, you can copy the chart version. The chart is therefore insensitive as in your default values ‘values.yaml’ here nginx:stable

Reply:
It must be a semver